frontend/packages/kubevirt-plugin/src/k8s/helpers/vm-clone.ts

```diff
@@ -145,7 +145,45 @@ export class VMClone {
         const dvName = getVolumeDataVolumeName(volume);
         const dataVolume = dvLookup[`${this.oldVMNamespace}-${dvName}`];
 
-        if (dataVolume) {
+        // when we create DV with storage section, although we request 1Gi for example
+        // the actual PVC created with 1.06Gi~ and than state that DV size is smaller than PVC
+        // that's why we need to fetch PVCs and get actual size
+        const pvcSize = pvcs
+          ?.filter((pvc) => pvc?.metadata?.name === dvName)
+          ?.map((pvc) => pvc?.spec?.resources?.requests?.storage)
+          ?.join('');
+
+        if (dataVolume && dataVolume.spec?.storage) {
```

Comment:
Couldn't this just be shortened to `if (dataVolume?.spec?.storage) {`?

Reply:
thanks fixed